plugins/com.google.cloud.tools.eclipse.appengine.deploy.ui/helpContexts.xml

```diff
@@ -2,15 +2,15 @@
 <contexts>
   <context id="DeployAppEngineStandardProjectContext" title="App Engine Deployment - Standard Environment">
     <!-- Eclipse requires escaped pseudo-HTML here -->
-    <description>Properties used when deploying the project to the App Engine standard environment.
+    <description>Properties used when deploying the project to the App Engine Standard environment.
      
      &lt;b>Account&lt;/b>
      Google user account with permissions to deploy the project.
      
-     &lt;b>Project ID&lt;/b>
-     The customized name you chose when you created the Cloud Platform project in the Google Cloud Console.
+     &lt;b>Project&lt;/b>
+     The Google Cloud Platform project that will be the target for the deployment. If you want to create a new project, you can click on the link to open the Cloud Console.
```

Comment:
Present tense, "The Google Cloud Platform project that is the deployment target" 
"you can click on the link to open the Cloud Console" --> "follow the link open the Cloud Console"